simplicity-sys/depend/env.c

```diff
@@ -0,0 +1,68 @@
+#include <stdlib.h>
+#include "simplicity/elements/env.h"
+#include "simplicity/primitive/elements/primitive.h"
+
+const size_t c_sizeof_rawBuffer = sizeof(rawBuffer);
+const size_t c_sizeof_rawOutput = sizeof(rawOutput);
+const size_t c_sizeof_rawInput = sizeof(rawInput);
+const size_t c_sizeof_rawTransaction = sizeof(rawTransaction);
+const size_t c_sizeof_rawTapEnv = sizeof(rawTapEnv);
+const size_t c_sizeof_txEnv = sizeof(txEnv);
```

Comment:
Since we are using C11 we can also use `alignof` and assert the alignment.

Reply:
Done